py/torch_tensorrt/dynamo/conversion/impl/select.py

```diff
@@ -1,16 +1,24 @@
-from typing import Optional, cast
+import logging
+from typing import Optional, Sequence, Union, cast
 
 import numpy as np
+import tensorrt as trt
 from torch.fx.node import Target
 from torch_tensorrt.dynamo._SourceIR import SourceIR
+from torch_tensorrt.dynamo.conversion.converter_utils import broadcastable
+from torch_tensorrt.dynamo.conversion.impl.elementwise import convert_binary_elementwise
 from torch_tensorrt.dynamo.conversion.impl.shape import get_shape_with_dynamic_shape
 from torch_tensorrt.fx.converters.converter_utils import (
     get_positive_dim,
+    get_trt_tensor,
```

Comment:
Switch to use the `from torch_tensorrt.dynamo.conversion.converter_utils` implementation